modules/assisted-installer-configuring-networking.adoc

```diff
@@ -0,0 +1,51 @@
+// This is included in the following assemblies:
+//
+// assisted-installer-installing.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-networking_{context}"]
+= Configuring networking
+
+Before installing {product-title}, you can configure the cluster network.
+
+.Procedure
+
+. In the *Networking* page, select one of the following if it is not already selected for you:
++
+** *Cluster-Managed Networking*
++
+** *User-Managed Networking*
```

Comment:
Do we want to say what these options mean?

Reply:
definitely. and what are the expectations from the users.
We can point/copy relevant sections from upi/ipi docs